### ISSUE TITLE
change non-Syriaca bibl links from tei:ptr to tei:ref 

### ISSUE THREAD

**Issue by wlpotter** [srophe/bethqatraye-data#71]
See https://github.com/srophe/srophe-app-data/issues/781

**Comment by wlpotter** [comment]
Note to self: data changes currently being made here: https://github.com/srophe/bethqatraye-data/tree/issue-71

**Comment by wlpotter** [comment]
The data on [this branch](https://github.com/srophe/bethqatraye-data/tree/issue-71) is ready to be pulled in. Waiting until the schema is updated (see https://github.com/srophe/srophe-app-data/issues/781).

Will need to upload the data manually.

**Comment by wlpotter** [comment]
@wsalesky FYI this was a batch change, so the data will need to be uploaded manually

**Comment by wlpotter** [comment]
We may revert these changes pending a discussion of whether or not we want to maintain a distinction between Syriaca and non-syriaca bibl uris.

**Comment by wlpotter** [comment]
Decision: We have decided not to change how Syriaca and non-Syriaca bibl URIs are handled. Both will be encoded as `tei:ptr/@target` attribute values. (See further discussion [here](https://github.com/srophe/srophe-app-data/issues/781#issuecomment-628162588)).

@wsalesky No data will have changed, since I've reverted the branch merge. However, this process looks to have taken down the webhooks. We will be making a few other batch changes to the data, but we are almost to a point where we will lock down any further changes and leave the data static. At that point we will likely want to do a final upload of the data and then disable the webhooks. We can check in about this next week.
